src/lib.rs

```diff
@@ -120,3 +121,16 @@ pub async fn listen<A: Into<String>, R: Into<PathBuf>>(
         rx,
     })
 }
+
+/// Configure live-server to always hard reload the page instead of hot-reload
+/// ```
+/// use live_server::{listen, hard_reload};
+///
+/// async fn serve_hard() -> Result<(), Box<dyn std::error::Error>> {
+///     hard_reload();
```

Comment:
`ROOT` and `TX` are `OnceCell` so `start()` can really only be called once. I agree it's counterintuitive to have a sparate function for reload mode, maybe something like this?
```rust
listen("127.0.0.1:8080", "./").await?.hard_reload().start().await;
```

Reply:
> ROOT and TX are OnceCell so start() can really only be called once.

Didn't realize this before. Maybe it can be improved in the future. This `hard_reload()` issue can be put aside for now, I will try to improve it later.